ocs_ci/ocs/ui/base_ui.py

```
@@ -1247,9 +1247,38 @@ def _set_driver(cls) -> WebDriver:
+                    or os.getenv("https_proxy")
+                )
+                no_proxy = (
+                    config.ENV_DATA.get("no_proxy")
+                    or os.getenv("NO_PROXY")
+                    or os.getenv("no_proxy")
+                )
+
+                if http_proxy or https_proxy:
+                    # Use HTTPS proxy if available, otherwise HTTP proxy
+                    proxy_server = https_proxy if https_proxy else http_proxy
+                    logger.info(f"Configuring proxy ({proxy_server}) for browser")
```

Ruff (0.16.3)

[warning] Logging statement uses f-string

(G004)